apps/src/templates/sectionProgress/progressTables/progressTableHelpers.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classnames from 'classnames';
+import moment from 'moment';
+
+// styles rows: dark backgrounds, primary rows, expanded rows
+export function progressTableRowWrapper(rowData, formattedRow) {
```

Comment:
the naming here is a little misleading. you are formatting a _cell_ based on row properties. it would be awesome to be able to format a whole row at once (as we were previously doing with `tr`), but we can't, and yet this naming makes it sounds like that's what we're doing. i propose renaming the function to `progressTableCellWrapper` or `progressTableRowCellWrapper` and the prop to `renderedCell`.

Reply:
Removed this function in favor of `onRow`